inc/taxonomies.php

```diff
@@ -1645,6 +1670,48 @@ public function taxopress_manage_taxonomies()
 <input type="hidden" name="cpt_tax_status" id="cpt_tax_status"
    value="<?php echo esc_attr($tab); ?>"/>
 </p>
+                        <?php
+                        if($taxonomy_edit){
```

Comment:
@jonathanpaulbbaroma This whole section was removed in another PR (https://github.com/TaxoPress/TaxoPress/pull/1455) but it's part of your commit. 

Always make sure you work on updated development branch and when making commit, only commit the changes you made and no addition or subtraction of codes that are not part of the requirement.